### DISCUSSION TITLE
Where to find v1.165.2?

### DISCUSSION THREAD

**Discussion by JakobTischler** [p42ai/refactor-vscode#60] [Q&A] [answered]
Hello,

in the extension panel it says 

> If you want to keep using the free features without upgrading, you can continue to use the JS Assistant v1.165.2, which will remain available in the marketplace.

Could someone please provide a link to that extension version? I can't for the life of me find it (https://marketplace.visualstudio.com/publishers/p42ai).

Thanks.

**Comment by lgrammel** [comment, marked as answer]
Hi Jakob,

here's a quick guide on how to install an older version:

https://code.visualstudio.com/updates/v1_30#_install-previous-versions

I'll post links to the VSIX file later when I'm at my computer.

Best, Lars

**Reply by lgrammel** [reply]
Here's a link to down the VSIX: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/p42ai/vsextensions/refactor/1.165.2/vspackage

**Comment by JakobTischler** [comment]
Awesome, thank you for the quick answer. And also this extension in general.

Grüße aus Kreuzberg
